ironfish/src/utils/math.ts

```diff
@@ -14,6 +14,22 @@ function arrayAverage(values: number[]): number {
   return total / values.length
 }
 
+function arrayMedian(values: number[], isSorted = false): number {
+  if (values.length === 0) {
+    return 0
+  }
+
+  // TODO(mat): We can use values.toSorted() when we set NodeJS min version to 20
```

Comment:
Thinking out loud, seems worth to expedite bumping the min version.

Reply:
So far, we've been following the LTS releases of Node: https://nodejs.dev/en/about/releases/

Which means, we'll be upgrading to 20 in October! 🎉